.gitignore

```diff
@@ -9,4 +9,4 @@ tsconfig.tsbuildinfo
 **/**/coverage/
 **/**/.nyc_output/
 .DS_Store
-*.xml
+test-results.*.xml
```

Comment:
QQ: Why was this changed? Were there any `.xml` files which were wrongly ignored?